src/command_modules/azure-cli-network/azure/cli/command_modules/network/commands.py

```diff
@@ -349,6 +351,28 @@ def _make_singular(value):
                                      custom_func_name='update_ag_{}'.format(_make_singular(subresource)),
                                      child_collection_prop_name=subresource, validator=create_validator)
 
+    with self.command_group('network application-gateway rewrite-rule', network_ag_sdk, min_api='2018-12-01') as g:
```

Comment:
Not sure if this is a major issue or relevant. But rewrite rules are supported in `2018_10_01`. However, conditions are added in `2018-12-01`